clients/drcachesim/CMakeLists.txt

```diff
@@ -206,6 +206,7 @@ add_exported_library(drmemtrace_simulator STATIC
   simulator/policy_fifo.cpp
   simulator/policy_lfu.cpp
   simulator/policy_lru.cpp
+  simulator/policy_rrip.cpp
```

Comment:
Please add a unit test for RRIP in clients/drcachesim/tests/cache_replacement_policy_unit_test.cpp.